cfme/base/ui.py

```diff
@@ -1084,3 +1084,54 @@ def step(self):
         from cfme.automate import automate_menu_name
         self.prerequisite_view.navigation.select(
             *automate_menu_name(self.obj.appliance) + ['Simulation'])
+
+
+class AutomateImportExportBaseView(BaseLoggedInPage):
+    flash = FlashMessages('div.import-flash-message')
+    title = Text('.//div[@id="main-content"]//h1')
+
+    @property
+    def in_import_export(self):
+        from cfme.automate import automate_menu_name
```

Comment:
Actually, it is the same as in other places, I guess that is due to risk of circular imports.